src/content/core/user-roles-and-permissions.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: User Roles and Permissions in Okteto
+description: Use volume snapshots in your development environments
+sidebar_label: User Roles and Permissions
+id: user-roles-and-permissions
+---
+
+
+The Okteto platform offers a robust Role-Based Access Control (RBAC) system to manage and control access to different environments within the platform.
+This page outlines the roles, permissions, and key features available to Admins and Developers in Okteto.
+
+## User Roles
+
+In Okteto, there are two primary roles: **Admins** and **Developers**. These roles were previously known as “Admin” and “User,” but we have updated the terminology for clarity and consistency across our platform.
```

Comment:
I think mentions to previous behavior like `These roles were previously known as “Admin” and “User,” but we have updated the terminology for clarity and consistency across our platform.` don't work well on docs as they get updated. I think it's better to mention it as part of the release notes